tensorflow/python/ops/array_ops.py

```diff
@@ -1132,16 +1163,32 @@ def boolean_mask(tensor, mask, name="boolean_mask", axis=None):
   # 1-D example
   tensor = [0, 1, 2, 3]
   mask = np.array([True, False, True, False])
-  boolean_mask(tensor, mask)  # [0, 2]
+  tf.boolean_mask(tensor, mask).numpy()  
+  # ==> array([0, 2], dtype=int32)
```

Comment:
I think tensor does not have expose an api of `numpy()`? Maybe something like the following?
```
value = tf.boolean_mask(tensor, mask)
tf.Session().run(value)
# ==> array([0, 2], dtype=int32)
>>> 
```

Reply:
Added a note to both examples for `tf.boolean_mask` indicating that the code should be run in eager mode.